### DISCUSSION TITLE
autostart not working on macosx m1

### DISCUSSION THREAD

**Discussion by karawitan** [utmapp/UTM#6606] [UTM Help (macOS)]
**Describe the issue**  
I'd like to autostart a vm at login on macosx Sonoma (14.5). Already tried all from #4179 

I tried to :
- put my vm package to login items
- put both my vm package to login items, as long as /Applications/UTM.app
- type open utm://start?name=my_vm   in a Terminal
- browse to utm://start?name=my_vm with a few browsers (firefox,yandex)
- same with utm://start?name=<vm identifier>
- same with utm://start?name=<vm UUID>

**Expected result**

The vm my_wm starts

**Result obtained**

UTM app opens. No special VM is even selected nor launched.

**Configuration**  
* UTM Version: 4.5.3(99)
* macOS Version: 14.5
* Mac Chip (Intel, M1, ...): M1 

**Crash log**  
N/A

**Debug log** 
no single line of output on /var/log/system.log when issue is reproduced

**Upload VM**

**Comment by karawitan** [comment]
ok as a workaround, I will use ``` /Applications/UTM.app/Contents/MacOS/utmctl start my_vm ```

**Comment by JHPWBN** [comment]
Hi, I have the same issue,. Can you please explain how you get he workaround into the list of apps to start at login ?
